proposals/0304-structured-concurrency.md

```diff
@@ -6,6 +6,54 @@
 * Status: **Active Review (March 31 - April 16 2021)**
 * Implementation: Available in [recent `main` snapshots](https://swift.org/download/#snapshots) behind the flag `-Xfrontend -enable-experimental-concurrency`
 
+## Table of contents
+
+* [Introduction](#introduction)
+* [Motivation](#motivation)
+* [Structured concurrency](#structured-concurrency-1)
+   * [Tasks](#tasks)
+   * [Child tasks](#child-tasks)
+   * [Partial tasks](#partial-tasks)
+   * [Executors](#executors)
```

Comment:
Should "partial task" be renamed to "job" in these two sections?

```suggestion
   * [Jobs](#jobs)
   * [Executors](#executors)
```

Reply:
Hmm. That's the term we use in the implementation nowadays. Sure, that makes sense. Thanks!